static/lang/en.json

```diff
@@ -2551,10 +2551,12 @@
         "PreciousMaterialInubrixDescription": "This pale, malleable metal's unusual molecular structure allows it to partially pass through iron and steel without touching them. While this property is useful for making weapons that bypass metal armor, inubrix is barely sturdier than lead. Even in an alloyed state, this skymetal is so fragile that it's difficult to use in crafting reliable shields and less than ideal for crafting armor.",
         "PreciousMaterialLabel": "Precious Material",
         "PreciousMaterialLowGrade": "Low-Grade",
+        "PreciousMaterialKeepStone": "Keep Stone",
+        "PreciousMaterialKeepStoneDescription": "The crowning achievement of Highhelm's millennia of metallurgical advancements and engineering is the beautiful alloy called keep stone. Despite its name, no the use of a highly guarded process that smelts together adamantine and lead, Highhelm's greatest crafters were able to develop a material with an appearance that more closely resembles marble than metal. Keep stone is only slightly weaker than adamantine alone, but with the incredible ability to disrupt magic. Any spell or magical effect targeting raw keep stone must succeed at a DC 5 flat check or the effect is lost.",
```

Comment:
Some missing words:
"Despite its name, no stone is used in the production of keep stone. Through the use of a highly guarded process that smelts together adamantine..."

Reply:
It's a whole other additional paragraph that would make it the longest description by far. We can add it when materials are journal entry pages.